sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatParticipant.java

```diff
@@ -21,6 +21,15 @@
     @JsonProperty(value = "id", required = true)
     private CommunicationUserIdentifier id;
 
+    /*
+     * Identifies a participant in Azure Communication services. A participant
+     * is, for example, a phone number or an Azure communication user. This
+     * model must be interpreted as a union: Apart from rawId, at most one
+     * further property may be set.
+     */
+    @JsonProperty(value = "identifier")
```

Comment:
We need to update ReadMe on how to use this identifier. We may update ChangeLog as well.

Reply:
@marche0133 this CommunicationIdentifierModel is an internal model that will be returned in the service response. It should be converted to the CommunicationIdentifier using the serializer. Since it is internal and we dont intend for customers to use it, we should not be including it on the README or CHANGELOG